snap/snapcraft.yaml

```diff
@@ -20,6 +20,7 @@ hooks:
 environment:
   JAVA_HOME: $SNAP/usr/lib/jvm/java-17-openjdk-amd64
   LOG_DIR: $SNAP_COMMON/var/log/kafka
+  CLASSPATH: $SNAP/opt/kafka/libs/*
```

Comment:
**todo** I don't think this is necessary. All Jars are already in `/opt/kafka/libs` and this is the standard PATH being used by Kafka

Reply:
To add context to that, `opt/kafka/libs/kafka-run-class.sh` uses:

```bash
base_dir=$(dirname $0)/..

# classpath addition for release
for file in "$base_dir"/libs/*;
do
  if should_include_file "$file"; then
    CLASSPATH="$CLASSPATH":"$file"
  fi
done
```

So, it should pick up `base_dir=/opt/kafka`, then adds all of `libs` to the path.

Reply:
updated